### DISCUSSION TITLE
First possible day to selected in calendar

### DISCUSSION THREAD

**Discussion by roman5527** [fomantic/Fomantic-UI#1352] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Can I set in calendar first pissible day ? 

Example : today is 3.3.2020 
first pissible day is 6.3.2020 . 

every day example +3 days. 

can I set it ? 

thans

**Comment by jamessampford** [comment, marked as answer]
Docs are usually a good place to look?

https://fomantic-ui.com/modules/calendar.html#minmax-date

**Comment by roman5527** [comment]
Hi, thanks . I cant find it. 
Thank you very mutch.

**Comment by lubber-de** [comment]
Closing, because @jamessampford pointed directly to a solution with example sourcecode
